plugins/woocommerce/changelog/add-54617_empty_all_product_screen_redirect_2

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add is_always_accessible method to Task class, to allow tasks to still be accessible when list is hidden.
```

Comment:
Added a second changelog to reflect the change in the `Task` class specifically.